src/lims/lims.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { LIMSClient } from './lims.client';
+import { LabTestResult } from 'src/common/interfaces/lab-test-result';
+
+@Injectable()
+export class LIMSService {
+    constructor(private readonly ohspClient: LIMSClient) {}
+
+    async findPatientById(id: string): Promise<LabTestResult> {
+        return await this.ohspClient.findPatientById(id);
```

Comment:
Just a slight variable naming thing, I think it should be reflective. ohspClient -> limsClient